apps/pdc-frontend/src/app/[locale]/(rootLayout)/search/[query]/page.tsx

```diff
@@ -1,3 +1,4 @@
+import { decode } from 'html-entities';
```

Comment:
Ik snap niet waarom deze nodig is, kun je dat uitleggen? Waar komen die HTML entities vandaan?

Reply:
Als je een woord met een enkel aanhalingsteken in het zoekveld typt, converteert de browser het naar &#39;, bijvoorbeeld don't wordt don&#39;t."

https://loket.digitaal.utrecht.nl/nl/search/don't